lambdas/service/app.py

```diff
@@ -870,6 +873,44 @@ def get_integrations():
                     body=json.dumps(body))
 
 
+@app.route(
+    '/maintenance/schedule',
+    methods=['GET'],
+    cors=True,
+    method_spec={
+        'summary': 'A maintenance schedule as an JSON object',
+        'tags': ['Auxiliary'],
+        'responses': {
+            '200': {
+                'description': fd('''
+                    This object may be hanceforth refered to as "the schedule"
+                    or `schedule`.
+                    The `start` time of an event is its `actual_start` if set,
+                    or its `planned_start` otherwise. The `end` time of an event
+                    is its `actual_end` if set, or its `start` plus
+                    `planned_duration` otherwise. All events in the schedule are
+                    sorted by their `start` time. No two events have the same
+                    `start` time. Each event defines an interval
+                    `[e.start, e.end)` and there is no overlap between these
+                     intervals.
+
+                     A pending event is one where `actual_start` is absent. An
+                     active event is one where `actual_start` is present but
+                     `actual_end` is absent. There can be at most one active
+                     event.
+                ''')
+            }
+        }
+    }
+)
+def get_maintenance_schedule():
+    service = MaintenanceService()
```

Comment:
`MaintenanceService` has a `cached_property` which is made pointless if we construct a new instance of the service for every request. 

Normally the service would be cached on the controller, which would be cached on the app. We don't have a controller here, but I still think it would e a good idea to find a way to reuse this instance.